docs/reference/classes.md

```diff
@@ -3,31 +3,27 @@ classDiagram
   direction TB
   
   AdminUser --> User : Is a Type of
-  Term --> Search : Represents a common phrase in a set of
+  Term --> SearchEvent : has many
 
   User --> Categorization : Creates a
   User --> Category : Proposes a
-  Batch --> Search : Loads and Processes
   Categorization --> Term : Includes a
   Categorization --> Category : Includes a
 
   class Term
-    Term
     Term: id
+    Term: +String phrase
     Term: calculate_certainty(term)
     Term: list_searches()
```

Comment:
Could we change this to `list_search_events()`, now that we've changed the name of that class? (Or just remove it; as you noted in our meeting this morning, it's effectively a convenience method.)

Reply:
I chose to remove it (at least for now) from the Class.